transcom/mymove
# B 20977 usmc moves search int

## [B-20977](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1001184)

## Summary

This PR adds USMC to the destination GBLOC section of the office users search results when moves' are affiliated with the marines.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create Customer with USMC
2. Create Move/Shipment complete and sign
3. Search for move code (any office user role)
4. Confirm destination GBLOC shows USMC/geographic GBLOC
    example: "USMC/KKFA"

### Frontend


## Screenshots
_**USMC added to detination GBLOC**_
<img width="917" alt="Screenshot 2024-10-04 at 12 44 46 AM" src="https://github.com/user-attachments/assets/02045430-5db3-42e6-9c15-a3347bf01039">
